packages/ethereum-storage/test/ethereum-storage.test.ts

```diff
@@ -373,15 +371,6 @@ describe('EthereumStorage', () => {
       );
     });
 
-    it('cannot read if ethereumMetadataCache.getDataIdMeta fail', async () => {
```

Comment:
test dropped, `getDataIdMeta` cannot fail

Reply:
Actually it could fail if the provided store for `Keyv` is not an in-memory store, but a Redis store or something else. But I guess we could let any Keyv exception errors bubble up the stack and kill the NodeJS process.

Keyv storage should be able to reconnect itself to its backend (in case of connection issues with the backend) and it should be transparent for us. If an exception happens, then it is critical, and there is nothing that can be done on our side, I think.